mos-platform/cpm65/bios.S

```diff
@@ -0,0 +1,20 @@
+; Sets up the BIOS entrypoint.
+;
+; © 2022 David Given
+; This file is part of the llvm-mos-sdk project and is redistributable under
+; the terms of the Apache 2.0 license. See the LICENSE file in the project root
```

Comment:
Can these files instead be licensed using the "Apache 2.0 license with LLVM exceptions"? This is what most of the rest of our SDK uses, as it allows the copies of the object files included with statically-linked user binaries to be redistributed without including a copy of the Apache license.

Reply:
Yes, totally. Done.